.github/workflows/python-app.yml

```diff
@@ -49,14 +49,14 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install flake8 pytest black isort codecov pytest-cov
+          pip install flake8==3.7.7 pytest black==19.10b0 isort==4.3.20 codecov pytest-cov
```

Comment:
Why have older versions here now?

Reply:
It's fine to use newer ones. Either way we need to fix the versions of these imports otherwise black and isort are often not compatible with each other and the requirements keep changing. So we either install the new ones and change `pre-commit-config.yaml` or do this

Reply:
Let's use the newer ones then.

Reply:
Okay I'll update them then to use the latest (stable) version